Bots/DotNet/Skills/Composer/ComposerSkillBotDotNet/ComposerSkillBotDotNet/ComposerSkillBotDotNet.dialog

```diff
@@ -0,0 +1,190 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "name": "ComposerSkillBotDotNet",
+    "description": "",
+    "id": "A79tBe"
+  },
+  "autoEndDialog": true,
+  "defaultResultProperty": "dialog.result",
+  "triggers": [
+    {
+      "$kind": "Microsoft.OnUnknownIntent",
+      "$designer": {
+        "id": "mb2n1u"
+      },
+      "actions": [
+        {
+          "$kind": "Microsoft.SendActivity",
+          "$designer": {
+            "id": "kMjqz1"
+          },
+          "activity": "${SendActivity_DidNotUnderstand()}"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnEventActivity",
+      "$designer": {
+        "id": "wq9yy0",
+        "name": "ActivityRouterDialog"
+      },
+      "actions": [
+        {
+          "$kind": "Microsoft.SetProperty",
+          "$designer": {
+            "id": "z2JNUs"
+          },
+          "value": "=router.GetHostUrl(settings.skillHostEndpoint)",
```

Comment:
Does this work? If yes, then probably change the scope to conversation instead of user (that's cleaner and it won't persist beyond the conversation)

Reply:
Hi @gabog, we added this to be able to access the _HostURL_ from some of the scenarios like _MessageWithAttachments_ and _Cards_.
We changed the property's scope to _conversation_.
Thanks!